### DISCUSSION TITLE
How can get the document counts related to taxonomies

### DISCUSSION THREAD

**Discussion by liusming** [OrchardCMS/OrchardCore#12297] [General]
How can I show the counter of the document corresponding to the taxonomies indiviually as following picture shows?

Methods including moduels, workflow, liquid template or coding are all appreciated.

Thanks in advance

![Screen Shot 2022-08-30 at 1 18 20 PM](https://user-images.githubusercontent.com/4685718/187356110-e2bed078-c3a8-4254-9454-eed526ed76a2.png)

**Comment by ns8482e** [comment]
You can create a query that  group byTermContentItemId and Count published ContentItem

**Reply by liusming** [reply]
Thanks for your advise. 
I am wondering how can I link the (liquid, SQL or GraphQL?) query with (Numnber, Text or any) field when contenitem published? Or if I do the query in every request of page the system loading seems too heavy.

Thanks in advance.
